app/controllers/EmailAddressQuestionForOrganisationController.scala

```diff
@@ -82,13 +82,17 @@ class EmailAddressQuestionForOrganisationController @Inject()(
         },
         value => {
 
-          val redirectUsers = hasValueChanged(value, EmailAddressQuestionForOrganisationPage, mode, request.userAnswers)
+          val determineRoute = (value, mode) match {
+            case (true, CheckMode) => false
+            case (false, CheckMode) => true
+            case  _ => false
+          }
```

Comment:
Can be simplified to
val determineRoute = (value, mode) match {
            case (false, CheckMode) => true
            case  _ => false
          }